### PR TITLE
Jsreds/add device is pined on release 2c

### PR DESCRIPTION
**Pull request by JSReds**
Resolves: #1095

Project: https://balena.fibery.io/Work/Project/Server-side-pagination-for-devices-Cycle-3-539

### CODE REVIEW

File: src/features/hostapp/hooks/target-hostapp.ts
Comment (thgreasi) [inline src/features/hostapp/hooks/target-hostapp.ts:1]: The changed in this file need to be reverted.

**Review by thgreasi — APPROVED** [review]
LGTM but let's wait for an extra approval on both oB & bC

**Review by otaviojacobi — APPROVED** [review]

**Pull request merged**
